src/main/java/org/mockito/internal/invocation/InvocationsFinder.java

```diff
@@ -128,7 +127,9 @@ public static Location getLastLocation(List<Invocation> invocations) {
     public static Invocation findPreviousVerifiedInOrder(
             List<Invocation> invocations, InOrderContext context) {
         LinkedList<Invocation> verifiedOnly =
```

Comment:
Let's refactor this to an `List<Invocation>` so that we can use `Collectors.toList()` like the rest of the usages.